source/application/AMPIC887DriverServer/GCS2Commands/AMGCS2GetCurrentPositionCommand.h

```diff
@@ -30,13 +31,8 @@ class AMGCS2GetCurrentPositionCommand : public AMGCS2Command
 	  * the last time the command was run. If the command has not been run, or was
 	  * not successful when last run, then an empty map is returned.
 	  */
-	QHash<AMGCS2::Axis, double> axisPositions();
+	AMPIC887AxisMap<double> axisPositions();
```

Comment:
This (and in fact all the get commands) are run only once in the initialization of the controller. All other times the values are obtained from the cache (and the caches are updated each time a set command is called).